frontend/app-development/features/SettingsModalButton/SettingsModal/components/Tabs/AbouTab/AboutTab.module.css

```
@@ -1,3 +1,47 @@
 .contentWrapper {
   max-width: 500px;
 }
+
+.createdFor {
+  display: flex;
+  align-items: center;
+  background-color: var(--fds-semantic-surface-neutral-subtle);
+  border-radius: 4px;
+  padding-inline: 12px;
+  padding-block: 20px;
+  margin-top: 10px;
+}
+
+.createdBy {
+  display: flex;
+  align-items: center;
+  margin-block: 10px;
+  color: var(--fds-semantic-text-action-active);
+}
+
+.label {
+  margin-top: 20px;
+}
+
+.ownerParagraph {
+  margin-left: 15px;
+  color: var(--fds-semantic-text-action-active);
+}
+
+.createdByParagraph {
+  margin-left: 10px;
+  color: inherit;
+}
+
+.createdByIcon {
+  font-size: 1.5rem;
+  color: inherit;
+}
+
+.avatar {
+  max-height: 2em;
+}
+
+.dateParagraph {
```

Hva tenker du om å ikke si noe om "HTML-elementet" i klassenavnet men heller et mer generisk navn? Så om noen trenger å endre det til en div, span eller noe annet i fremtiden så blir det ikke mismatch med kalssenavnet.
```suggestion
.createdDate {
```